### ISSUE TITLE
Trackpad face buttons axis mappings

### ISSUE THREAD

**Issue by PepeDuke** [ValveSoftware/steamvr_unreal_plugin#114]
https://imgur.com/a/9X1N8kM
Using settings above i expect that steam vr mapping would result in axis movment as with old system but instead default bindings in steam vr is the code blow whitch is both axis (MoveRight,MoveUp) bound to X (1.0) not matter if i add _X or _Y to any of these. I'm going to use Action mapping instead of axis to fix this, but i feel like this should work

<details><summary>bindings</summary><p>

```
			{
				"path": "/user/hand/right/input/trackpad",
				"mode": "scalar_constant",
				"inputs": {
					"value": {
						"output": "/actions/main/in/MoveRight axis"
					}
				}
			},
			{
				"path": "/user/hand/left/input/trackpad",
				"mode": "scalar_constant",
				"inputs": {
					"value": {
						"output": "/actions/main/in/MoveRight axis"
					}
				}
			},
			{
				"path": "/user/hand/right/input/trackpad",
				"mode": "scalar_constant",
				"inputs": {
					"value": {
						"output": "/actions/main/in/MoveRight axis"
					}
				}
			},
			{
				"path": "/user/hand/left/input/trackpad",
				"mode": "scalar_constant",
				"inputs": {
					"value": {
						"output": "/actions/main/in/MoveRight axis"
					}
				}
			},
			{
				"path": "/user/hand/right/input/trackpad",
				"mode": "scalar_constant",
				"inputs": {
					"value": {
						"output": "/actions/main/in/MoveUp axis"
					}
				}
			},
			{
				"path": "/user/hand/left/input/trackpad",
				"mode": "scalar_constant",
				"inputs": {
					"value": {
						"output": "/actions/main/in/MoveUp axis"
					}
				}
			},
			{
				"path": "/user/hand/right/input/trackpad",
				"mode": "scalar_constant",
				"inputs": {
					"value": {
						"output": "/actions/main/in/MoveUp axis"
					}
				}
			},
			{
				"path": "/user/hand/left/input/trackpad",
				"mode": "scalar_constant",
				"inputs": {
					"value": {
						"output": "/actions/main/in/MoveUp axis"
					}
				}
			},
```

</p></details>

**Comment by 1runeberg** [comment]
Hi @PepeDuke - Yeah you should be able to do this in the Action/Key Mappings if I'm understanding your intent correctly.

If you're wanting to do something like free movement with Vector2s, along with adding the _X & _Y in the Axis Action Names, you need to use the Trackpad X & Trackpad Y keys:
![image](https://user-images.githubusercontent.com/17371351/73009032-d7b6fe80-3e63-11ea-9f1e-3fa531cf6c58.png)

**Comment by PepeDuke** [comment]
As i understand Trackpad X & Trackpad Y is not press of a button on controller, but just touching it, that works correctly but not what i need. My thoughts behind mentioning the _X and _Y was that i was thinking it might force the axis to be Y or X but it does not.

**Comment by 1runeberg** [comment]
the mode should be scalar_constant if it's an axis, so you can tweak the constant itself (when it triggers) from the steamvr dashboard if you're not getting the result or sensitivity you need.

**Comment by PepeDuke** [comment]
Yes I can, but if it's not default it's useless as you can't expect non-dev person to change something it steamvr dashboard. also i don't see how this is [help wanted] tag as this is request, i already have it working with action mapping but it would be nice to do it trough axis mapping.

**Comment by 1runeberg** [comment]
Hi @PepeDuke - As a dev you can use the SteamVRBindings dashboard to update the default bindings generated by the plugin ("Replace Default Binding" button or "Export Binding File" if you want to do a manual overwrite of you project jsons). As the engine UI doesn't have all the features the new SteamVR Input system has and paradigms are different in some cases, tweaking values and doing more advanced settings such as chords via the SteamVR Input dashboard is unavoidable at this time.

Your users can likewise use the input dashboard to override any developer settings once your game is published and share them to your community of users. 

However, we can review in 4.25 and see if we can easily derive and differentiate intents from what the Engine's UI and Input framework exposes at that time, as well as see what our options are from the runtime, so we appreciate your request/feedback.

**Comment by PepeDuke** [comment]
Thank you

**Comment by Andyhdr** [comment]
So with the latest hotfix in 4.24 the trackpad touching isn´t giving any axis values anymore. I have set everything up in the input settings, but the value is always 0

**Comment by 1runeberg** [comment]
Hi @Andyhdr - How are you using the axis value in your game?

I've also opened a separate Issue for this: https://github.com/ValveSoftware/steamvr_unreal_plugin/issues/115

**Comment by 1runeberg** [comment]
Fixed in 4.26, thanks for reporting!
